insonmnia/worker/server.go

```diff
@@ -1124,8 +1165,7 @@ func getDescriptionForBenchmark(b *pb.Benchmark) (Description, error) {
 		return Description{}, err
 	}
 	return Description{
-		autoremove: true,
```

Comment:
why drop autoremove?

Reply:
because it's racy in terms of logs fetching